separate_file.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def make_shirt(size, text):
+    """Describe t-shirt"""
+    print(f"The t-shirt is size {size} and says \'{text}.\'")
+
```

Comment:
The Hound violation above is asking for an extra line here. All functions should have two blank lines after them.

Reply:
@paulghaddad Ok, I added a second blank line. Will merge.